src/bundle/Resources/public/js/scripts/admin.main.translation.update.js

```diff
@@ -0,0 +1,10 @@
+(function(global, doc) {
+    const updateMainTranslation = (event) => {
+        doc.querySelector('#main_translation_update_language_code').value = event.target.value;
```

Comment:
Was this ID generated on the backend? If it was generated on the backend, I think it should be passed via data-attribute to JS.